apps/class-solid/src/components/Experiment.tsx

```diff
@@ -50,13 +50,11 @@ export function AddExperimentDialog(props: {
 }) {
   const defaultPreset = findPresetByName();
   const initialExperimentConfig = createMemo(() => {
-    const config = defaultPreset.parse({});
     return {
       preset: "Default",
       reference: {
-        ...config,
+        ...defaultPreset.config,
```

Comment:
This use same config instance everywhere, we should clone it. `...` does a shallow copy, which means initialstate.h_0 is shared.